asset-transfer-basic/rest-api-typescript/src/config.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const logLevel = process.env.LOG_LEVEL || 'info';
+
+export const port = Number(process.env.PORT || 3000);
+
+export const asLocalHost = process.env.AS_LOCAL_HOST == 'true' || true;
+
+export const identityName = 'restServerIdentity';
+
+export const mspId = process.env.MSP_ID || 'Org1MSP';
+
+export const channelName = process.env.CHANNEL_NAME || 'mychannel';
+
+export const chaincodeName = process.env.CHAINCODE_NAME || 'basic';
+
+if (!('CONNECTION_PROFILE' in process.env)) {
+  throw new Error('CONNECTION_PROFILE environment variable not set');
+}
+export const connectionProfile = process.env.CONNECTION_PROFILE || '';
+
+if (!('CERTIFICATE' in process.env)) {
+  throw new Error('CERTIFICATE environment variable not set');
+}
+export const certificate = process.env.CERTIFICATE || '';
```

Comment:
Wondering why the cert/private key are pulled out - rather than referring to a SDK wallet?

Reply:
A couple of reasons: there isn't a wallet in the test network; and trying to put a wallet in an env var seems... less than ideal. (Could go wild and implement a cloud db wallet but that's not really the focus of the sample and I'm not a fan of wallets anyway!)